eng/Version.Details.xml

```diff
@@ -206,9 +206,9 @@
       <Uri>https://github.com/dotnet/xharness</Uri>
       <Sha>e045ce36ce5d46bbd44097592bcf1b69c5ca75a2</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.PackageTesting" Version="7.0.0-beta.21463.4">
+    <Dependency Name="Microsoft.DotNet.PackageTesting" Version="7.0.0-beta.21472.6">
```

Comment:
The darc tool updates this file, so it's a black box to me :-).

Reply:
Usually many dependencies come from 1 build - like Arcade produces about 12 packages that get consumed by runtime and then this build is "flown" to runtime which means all those package will get updated together.

You don't update this file manually - either the dependency PR makes that happen or you call `darc update-dependencies` locally so I wouldn't worry about it much. If someone does that in their branch like Steve has, it usually happens that main gets up-to-date before we close the PR and then Steve would just revert changes here. Also, Arcade has to be validated first before we can bump it in runtime so probably we wouldn't even let Steve bump it and wait for main to catch up.

I don't understand how this file gets populated with new dependencies though, so no idea about the exact order. It's very probable that if you re-order this, it will get re-ordered back because it depends on some order in the maestro database. Or it will just stay how you set it. However, you will have a nasty conflict with whatever dependency PR is currently open.